src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -1521,9 +1534,6 @@ int ofi_am_post_recv(int vni, int nic)
 {
     int mpi_errno = MPI_SUCCESS;
 
-    /* Only root ctx for now */
-    MPIR_Assert(vni == 0 && nic == 0);
```

Comment:
Obviously this PR covers the multi-Vic case. What about multi-nic? Is the plan to continue only using NIC 0 for AM? Should this assertion stay to indicate that?

Reply:
You mean `MPIR_Assert(nic == 0);`? Probably.

Reply:
fixed.